### PR TITLE
Renamed tests for cli/test_scparam.py.

### PR DESCRIPTION
**Pull request by omaciel**
Test case naming follows #3038 conventions.

### CODE REVIEW

**Comment by omaciel** [comment]
```shell
(robottelo)┌[omaciel☮Ogs-MacBook-Pro.local]-(~/hacking/robottelo)-[git://rename-tests-cli-scparam ✔]-
└> py.test -v tests/foreman/cli/test_scparam.py --collect-only
================================================================= test session starts ==================================================================
platform darwin -- Python 2.7.10, pytest-2.8.3, py-1.4.31, pluggy-0.3.1 -- /Users/omaciel/.virtualenvs/robottelo/bin/python
cachedir: .cache
rootdir: /Users/omaciel/hacking/robottelo, inifile:
collected 1 items
<Module 'tests/foreman/cli/test_scparam.py'>
  <UnitTestCase 'SmartClassParameterTestCase'>
    <TestCaseFunction 'test_positive_list'>

===================================================================  in 0.28 seconds ===================================================================
```

File: tests/foreman/cli/test_scparam.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def run_puppet_module(self):
 
     @run_only_on('sat')
     @tier1
-    def test_bugzilla_1047794(self):
+    def test_positive_list(self):
         """@Test: Check if SmartClass Paramter Info generates an error
 
         @Feature: SmartClass Paramter - Info
```
Comment (sghai) [inline tests/foreman/cli/test_scparam.py]: I think `@Test` and `@Feature` description needs to be updated with `List` instead of `Info`

**Comment by sghai** [comment]
ACK, pending a non-blocking comment.

**Comment by omaciel** [comment]
@sghai good catch! Updated.

File: tests/foreman/cli/test_scparam.py [hunk 2]
```diff
@@ -20,12 +20,12 @@ def run_puppet_module(self):
 
     @run_only_on('sat')
     @tier1
-    def test_bugzilla_1047794(self):
-        """@Test: Check if SmartClass Paramter Info generates an error
+    def test_positive_list(self):
+        """@Test: Check if SmartClass Paramter `list` generates an error
```
Comment (elyezer) [inline tests/foreman/cli/test_scparam.py]: Inline code in rst is `` instead of `

File: tests/foreman/cli/test_scparam.py [hunk 3]
```diff
@@ -20,12 +20,12 @@ def run_puppet_module(self):
...
 
-        @Feature: SmartClass Paramter - Info
+        @Feature: SmartClass Paramter
 
-        @Assert: SmartClass Paramter Info does not generate an error
+        @Assert: SmartClass Paramter `list` does not generate an error
```
Comment (elyezer) [inline tests/foreman/cli/test_scparam.py]: Inline code in rst is `` instead of `

**Comment by elyezer** [comment]
ACK, pending comment.

**Comment by omaciel** [comment]
@elyezer updated.

**Comment by oshtaier** [comment]
ACK

**Pull request merged**